package.json

```diff
@@ -20,21 +20,25 @@
         "inquirer": "^7.3.3",
         "json-schema-faker": "^0.5.0-rc15",
         "mkdirp": "^1.0.4",
-        "node-fetch": "^2.6.1",
-        "wot-typescript-definitions": "^0.7.4-SNAPSHOT.1",
+        "wot-typescript-definitions": "0.7.4-SNAPSHOT.1",
         "xmlhttprequest": "^1.8.0"
     },
     "devDependencies": {
+        "@typescript-eslint/eslint-plugin": "^5.54.0",
+        "@typescript-eslint/parser": "^5.54.0",
         "chai": "^4.2.0",
+        "eslint": "^8.35.0",
         "mocha": "^6.2.0",
         "prettier": "2.0.5",
-        "typescript": "3.9.7"
+        "typescript": "4.9.5"
     },
     "scripts": {
         "build": "tsc",
-        "start": "node dist/wot-test-bench.js",
+        "start": "node dist/src/wot-test-bench.js",
```

Comment:
it should not generate it here. Could you fix tsconfig such that it is only under dist and not another level of src?

Reply:
Fixed at [f89dd79](https://github.com/tum-esi/testbench/pull/47/commits/f89dd7914846e9e0109569a6272fe941aea2f898)